src/Frontend/src/views/throughputreport/ThroughputSupported.vue

```diff
@@ -0,0 +1,59 @@
+<script setup lang="ts">
+import ConditionalRender from "@/components/ConditionalRender.vue";
+import { useIsSupported } from "@/composables/serviceSemVer";
+import { environment } from "@/composables/serviceServiceControl";
+import { computed } from "vue";
+
+const minimumSCVersionForThroughput = "5.0.0";
+const isThroughputSupported = computed(() => useIsSupported(environment.sc_version, minimumSCVersionForThroughput));
+</script>
+
+<template>
+  <ConditionalRender :supported="isThroughputSupported">
+    <template #unsupported>
+      <div class="text-center monitoring-no-data">
+        <h1>System Throughput</h1>
+        <p>
+          The minimum version of ServiceControl required to enable the System Throughput feature is
+          <span> {{ minimumSCVersionForThroughput }} </span>.
+        </p>
+        <div>
+          <a class="btn btn-default btn-primary" href="https://particular.net/downloads" target="_blank">Update ServiceControl to latest version</a>
+        </div>
+      </div>
+    </template>
+    <slot />
+  </ConditionalRender>
+</template>
+
+<style scoped>
+.monitoring-no-data {
+  margin: 60px auto 120px;
```

Comment:
can all the px sizing, unless it's for something like a border width, be changed to em sizing?